lib/handlers/injector-sor.ts

```diff
@@ -188,10 +188,11 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
               break
           }
 
-          let provider: StaticJsonRpcProvider
+          let provider
```

Comment:
is there a type we can assign here?

Reply:
I realized I can leave the type declaration as is, and do typecasting later